app/Http/Controllers/Admin/DiagramController.php

```diff
@@ -144,20 +148,21 @@ public function saveLocation(Request $request, $id)
 
     public function saveAjaxLocation(Request $request)
     {
-        $data = $this->locationRepository->create([
-            'name' => $request->name,
-            'workspace_id' => $request->id,
-            'color' => $request->color,
+        $this->validate($request, [
+            'id' => 'required',
+            'seat' => 'required'
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/DiagramController.php</br>**Line**: 153</br>Each line in an array declaration must end in a comma